doc/install_doc/installation.rst

```diff
@@ -134,14 +134,27 @@ compatible. This includes Moodle, edX, among many others.
 
 .. _LTI: http://www.imsglobal.org/LTI/v1p1/ltiIMGv1p1.html
 
+It is recommended to create a folder for INGInious and subfolders for tasks and backup, e.g.:
+::
+
+    $ mkdir -p /var/www/inginious
+    $ cd /var/www/inginious
+    $ mkdir tasks
+    $ mkdir backup
+
 To configure the web app automatically, use the ``inginious-install`` CLI.
 
 ::
 
     $ inginious-install
 
-This will help you create the configuration file in the current directory. For manual configuration and details, see
-:ref:`ConfigReference`.
+This will help you create the configuration file in the current directory. 
+When asked about the tasks folder, enter an absolute folder: /var/www/inginiuos/tasks .
```

Comment:
Looks way much better ! There's just a small typo here : ``inginious``. I'll be able to merge it afterwards.

Reply:
Fixed